spec/models/manageiq/providers/azure/cloud_manager/provision_workflow_spec.rb

```diff
@@ -143,8 +143,8 @@
                                           :supports_64_bit => true)
         ems.flavors << FactoryGirl.create(:flavor, :name => "Standard_A1", :supports_32_bit => false,
                                           :supports_64_bit => true)
-        ems.resource_groups << FactoryGirl.create(:resource_group)
-        ems.resource_groups << FactoryGirl.create(:resource_group)
+        ems.resource_groups << FactoryGirl.create(:resource_group, :type => 'ManageIQ::Providers::Azure::ResourceGroup')
+        ems.resource_groups << FactoryGirl.create(:resource_group, :type => 'ManageIQ::Providers::Azure::ResourceGroup')
```

Comment:
Can you create a separate factory file for this Azure ResourceGroup?

Similar to how it's done in Amazon for [`cloud_volume_amazon`](https://github.com/ManageIQ/manageiq-providers-amazon/blob/27cb9926a6432401def82d312159fa6d8520ddae/spec/factories/cloud_volume.rb#L2-L3)